bigbluebutton-html5/imports/ui/components/nav-bar/queries.ts

```diff
@@ -0,0 +1,37 @@
+import { gql } from '@apollo/client';
+
+export const TALKING_INDICATOR_SUBSCRIPTION = gql`subscription UserVoice($limit: Int!, ) {
+    user_voice(where: {showTalkingIndicator: {_eq: true}}, order_by: [{startTime: desc_nulls_last}, {endTime: desc_nulls_last}], limit: $limit) {
+        user {
+          name
+          color
+          userId
+        }
+        floor
+        spoke
+        joined
+        muted
+        startTime
+        talking
+      }
+  }`;
+
+export const MEETING_IS_BREAKOUT = gql ` subscription MeetingIsBreakout {
+    meeting {
+      isBreakout
+    }
+  }`;
+
+  export const AM_I_MODERATOR = gql ` subscription AmIModerator{
+    user {
```

Comment:
Request user info from endpoint `user_current` instead of `user`.

Reply:
Done!